docs/modules/getting_started/examples/code/getting_started.sh

```diff
@@ -22,18 +22,18 @@ helm repo add stackable-dev https://repo.stackable.tech/repository/helm-dev/
 # end::helm-add-repo[]
 echo "Installing Operators with Helm"
 # tag::helm-install-operators[]
-helm install --wait commons-operator stackable-dev/commons-operator --version 0.3.0-nightly
+helm install --wait commons-operator stackable-dev/commons-operator --version 0.4.0-nightly
```

Comment:
I think something went awry when we tagged the docs as the `docs_templating.sh` script was not run, so we have https://github.com/stackabletech/spark-k8s-operator/blob/docs/0.5/docs/modules/getting_started/examples/code/getting_started.sh#L25-L27. The nightly versions should be replaced with stable ones, the changes tagged with docs/XX, and then revewrt to the nightly versions.